From jhovold@gmail.com  Tue May 11 16:27:32 2010
From: Johan Hovold <jhovold@gmail.com>
Date: Fri,  7 May 2010 19:58:32 +0200
Subject: tty: fix obsolete comment on tty_insert_flip_string_fixed_flag
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: linux-kernel@vger.kernel.org, Johan Hovold <jhovold@gmail.com>
Message-ID: <1273255112-30671-1-git-send-email-jhovold@gmail.com>


Comment was not updated when tty_insert_flip_string was generalised.

Signed-off-by: Johan Hovold <jhovold@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/tty_buffer.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/char/tty_buffer.c
+++ b/drivers/char/tty_buffer.c
@@ -238,7 +238,7 @@ EXPORT_SYMBOL_GPL(tty_buffer_request_roo
  *	@size: size
  *
  *	Queue a series of bytes to the tty buffering. All the characters
- *	passed are marked as without error. Returns the number added.
+ *	passed are marked with the supplied flag. Returns the number added.
  *
  *	Locking: Called functions may take tty->buf.lock
  */
